src/common/components/hosts/utils.ts

```diff
@@ -52,7 +52,8 @@ export const canReset = (clusterStatus: Cluster['status'], status: Host['status'
   ['adding-hosts'].includes(clusterStatus) &&
   ['error', 'installing-pending-user-action'].includes(status);
 
-export const canEditRole = (cluster: Cluster): boolean => !isSNO(cluster);
+export const canEditRole = (cluster: Cluster): boolean =>
```

Comment:
I believe that `canEditRole` should equal to `canEditHost` because a host role can be edited when a host can be edited. so I'd go for
```
export const canEditRole = canEditHost
```

Also this way it would take cluster status and host status as parameters which makes it more reusable in CIM as well.

Reply:
Makes sense

Reply:
I think it's a good solution

Reply:
The PR with the changes: https://github.com/openshift-assisted/assisted-ui-lib/pull/1590